.idea/runConfigurations/AllCOMPTests.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
```

Comment:
Why do we need these xml files which are under .idea/runConfigurations/*.xml

Reply:
I like to include them so that its easy for us to run a certain subset of tests to double check any changes work with our scenarios.